lib/LibraryTemplatePlugin.js

```diff
@@ -34,47 +42,61 @@ LibraryTemplatePlugin.prototype.apply = function(compiler) {
 	compiler.plugin("this-compilation", function(compilation) {
 		switch(this.target) {
 			case "var":
-				compilation.apply(new SetVarMainTemplatePlugin("var " + accessorAccess(false, this.name)));
+				var varExpressions = this.names.map(function(name) {
```

Comment:
https://github.com/webpack/webpack/pull/2984/commits/473ad170ec009018705a5c49e3c230197ce30a0b should solve this